am-lib/src/test/java/uk/gov/hmcts/reform/amlib/helpers/DefaultRoleSetupDataFactory.java

```diff
@@ -0,0 +1,42 @@
+package uk.gov.hmcts.reform.amlib.helpers;
+
+import com.fasterxml.jackson.core.JsonPointer;
+import javafx.util.Pair;
+import uk.gov.hmcts.reform.amlib.enums.Permission;
+import uk.gov.hmcts.reform.amlib.enums.SecurityClassification;
+import uk.gov.hmcts.reform.amlib.models.DefaultPermissionGrant;
+
+import java.util.Map;
+import java.util.Set;
+import java.util.concurrent.ConcurrentHashMap;
+
+import static uk.gov.hmcts.reform.amlib.helpers.TestConstants.READ_PERMISSION;
+import static uk.gov.hmcts.reform.amlib.helpers.TestConstants.RESOURCE_NAME;
+import static uk.gov.hmcts.reform.amlib.helpers.TestConstants.RESOURCE_TYPE;
+import static uk.gov.hmcts.reform.amlib.helpers.TestConstants.ROLE_NAME;
+import static uk.gov.hmcts.reform.amlib.helpers.TestConstants.SERVICE_NAME;
+
+public class DefaultRoleSetupDataFactory {
+
+    private DefaultRoleSetupDataFactory() {
+        //NO-OP
```

Comment:
Some people do `throw new UnsupportedOperationException("Constructing utility class is not supported");`. I think it is more obvious than using cryptic "NO-OP". Shall we start doing it?